examples/k8s.yaml

```diff
@@ -71,13 +73,9 @@ provision:
       kubectl apply -f https://raw.githubusercontent.com/flannel-io/flannel/v0.14.0/Documentation/kube-flannel.yml
       # Control plane node isolation
       kubectl taint nodes --all node-role.kubernetes.io/master-
-  - mode: user
-    script: |
-      #!/bin/bash
-      set -eux -o pipefail
-      mkdir -p $HOME/.kube
-      sudo cp -f /etc/kubernetes/admin.conf $HOME/.kube/config
-      sudo chown $(id -u):$(id -g) $HOME/.kube/config
+      sed -e '/server:/ s/192.168.5.15/127.0.0.1/' -i $KUBECONFIG
```

Comment:
Side comment, outside the scope of this PR:

This reminds me that we really should source `lima.env` into the provisioning scripts, so we can use e.g. `LIMA_CIDATA_SLIRP_IP` for the address here. But we don't even have the have that symbolic name yet.

Reply:
I just realized that this is a `system` provisioning script, so already has all `lima.env` settings sourced. I've created #518 to define a name for the ip address.

Reply:
@afbjorklund Since #518 has been merged now, maybe rebase and use `LIMA_CIDATA_SLIRP_IP_ADDRESS` in this PR?

Reply:
Sure. Ironically, noted another error: `/mnt/lima-cidata/provision.system/00000001: line 12: HOME: unbound variable`